android/src/main/java/io/radar/capacitor/RadarPlugin.java

```diff
@@ -128,77 +140,99 @@ public void onLog(@NonNull Context context, @NonNull String message) {
     @Override
     public void load() {
         sPlugin = this;
+        String publishableKey = getContext().getString(R.string.radar_publishableKey);
```

Comment:
Thinking through how `load()` would interact with `initialize()`.  I guess existing apps would end up calling `Radar.initialize()` twice now - once in `load()` and once manually calling `Radar.initialize()` from `MainActivity` (https://radar.io/documentation/sdk/capacitor#install).

Wonder how much we should be concerned with duplicate calls to `Radar.initialize()`. Looks like some care is taken to only initialize underlying data one time (ie, safe to call `Radar.initialize()` twice), though there are a few things that would run twice.

https://github.com/radarlabs/radar-sdk-android/blob/f4f5011fe2f88ae5d7c02e6570492cd2d06189a4/sdk/src/main/java/io/radar/sdk/Radar.kt#L365

It does seem like a good direction to go in so that people don't have to write android / ios code to manually call `Radar.initialize()`

cc @brettguenther @jsani-radar @nickpatrick for more product thoughts on this

Reply:
Maybe this just becomes a 1) docs and 2) migration guide thing?

Reply:
Yeah I was thinking this would go in the migration guide. The way the docs work right now don't say anything about calling the `registerPlugin(RadarPlugin.class)` method, which I believe is the _proper_ way to load a capacitor plugin (from the docs I read). My guess is that right now, onLoad is not getting called with the documentation as-is (just calling `initialize()`) - in which case migrating to the new approach would be a replacement (also, if following capacitor docs instead of Radar docs, it might be more straight-forward)

Reply:
Gotcha, sounds good. 👍   We can add this stuff to the migration guide and the docs.